### ISSUE TITLE
How to display latex math on our results pages?

### ISSUE THREAD

**Issue by cdcapano** [ligo-cbc/pycbc#946]
The table summary code in pycbc/inference can output parameter labels and values in LaTeX, to allow for more human-readable results on our results pages. To get these to render correctly, we need some way to convert LaTeX to html.

Currently, we have a function in `str_utils` (`latex_to_html`) that tries to do latex to html conversion, but I don't think this is going to work. Already, I'm running into issues where this function is causing some things to display in-correctly (for instance, displaying "$\mathcal{M} (\mathrm{M}_{\odot}$" for chirp mass won't work). I think it'll be too much work to try to write a parser that can deal with all the possible latex.

Most web sites use the MathJax CDN to render latex math. However, @cmbiwer said CDNs do not always work on LDG sites (see PR #898). I looked into trying to add the MathJax Java scripts to pycbc's scripts directory, but they appear to be too big. Even stripping it down to the bare essentials, the directory is still a ~3MB. It might be possible to cut it down further, but I doubt I'd be able to get it under 1MB, since there is a fonts directory that needs to be included to get things to render correctly.

The simplest thing right now would be to just use the MathJax CDN. @cmbiwer What was the issue you saw using it? All of the links I've seen seem to render correctly.

**Comment by cmbiwer** [comment]
So I did a test with MathJax and it doesn't seem to have the same security problem that the other CDN's I've used before did at the sites, ie. it rendered for other users. So my initial concern doesn't seem to be a problem here.

I think like a week or so ago, I also stripped down the library and it is fairly large as you say.

To be honestly though I think that table just looks really ugly now and mismatched with the rest of the page. Particularly I'm talking about displaying the numbers with the MathJax.

I also don't think the LaTeX we need display needs to be all that complicated. Maybe a handful of Greek letters and some subscripts. If you do a replace it picks up the HTML CSS and looks much nicer.

It seems that I'm in the minority here so if you want to use MathJax then delete the LaTeX -> HTML function and go ahead, I guess in the scheme of things its not a big deal. I'll call it retro 90's and move on.
